lib/response.js

```diff
@@ -229,7 +233,16 @@ res.send = function send(body) {
     this.end();
   } else {
     // respond
-    this.end(chunk, encoding);
+    if (isChunkBlob) {
+      var response = this;
+      chunk.arrayBuffer().then(function (result) {
```

Comment:
Should we add a code path here to use https://nodejs.org/api/buffer.html#blobstream when supported? Not sure how usable that is, but seems Node.js is adding the ability to make a Blob from a file which could be huge.

Reply:
yes @dougwilson, you are right here for a high volume of data I used [`Writablestream`](https://nodejs.org/dist/latest-v20.x/docs/api/webstreams.html#class-writablestream) to achieve better control over this  
```javascript 
var WritableStream = require('stream/web').WritableStream;
        blob.stream().pipeTo(new WritableStream({
          write: function (chunk) {
            _this.write(chunk);
          },
          close: function () {
            _this.end();
          }
        }))
```